### ISSUE TITLE
(not issue) A new addon based on animbot

### ISSUE THREAD

**Issue by Irmitya** [quollism/blender-keyframetools#6]
I never knew about Animbot until this addon and I mostly don't get why I'd want it because of how it's info page explains it for Maya users and not just in General. 

This addon called AnimAide was made for 2.8, and you should take a look:
https://blender.community/c/today/Dhdbbc/
https://github.com/aresdevo/animaide

- - -- 
I never thought about messaging anyone on github but the way this one person did it was via an issue report.  Turns out github disabled Private messaging at some point.  Problem with this method is that it leaves an invalid mark in the history >_<

**Comment by quollism** [comment]
Yep know about this one. AnimAide is wayyyyy ahead of KFT in terms of its feature set, kind of makes me want to give up. :)
